src/frontend/hooks/useSendInnSkjemaTest/konverteringTilISøknadKontrakt.test.ts

```diff
@@ -20,58 +20,63 @@ describe('test konvertering fra ISøknad til ISøknadKontrakt', () => {
+        // const { input: iSøknad, output: expectedISøknadKontrakt } = testdata2;
+        const { input: iSøknad } = testdata2;
         spyOnUseApp(iSøknad);
         const { result } = renderHook(() => useSendInnSkjema(), {
             wrapper: TestProvidereMedEkteTekster,
         });
 
         const [_, formatert]: [boolean, ISøknadKontraktV8] = await result.current.sendInnSkjemaV8();
         expect(erGyldigISøknadKontrakt(formatert)).toBeTruthy();
-        expect(JSON.stringify(formatert)).toEqual(JSON.stringify(expectedISøknadKontrakt));
+        // expect(JSON.stringify(formatert)).toEqual(JSON.stringify(expectedISøknadKontrakt));
     });
     it('case 3', async () => {
-        const { input: iSøknad, output: expectedISøknadKontrakt } = testdata3;
+        // const { input: iSøknad, output: expectedISøknadKontrakt } = testdata3;
+        const { input: iSøknad } = testdata3;
         spyOnUseApp(iSøknad);
         const { result } = renderHook(() => useSendInnSkjema(), {
             wrapper: TestProvidereMedEkteTekster,
         });
 
         const [_, formatert]: [boolean, ISøknadKontraktV8] = await result.current.sendInnSkjemaV8();
         expect(erGyldigISøknadKontrakt(formatert)).toBeTruthy();
-        expect(JSON.stringify(formatert)).toEqual(JSON.stringify(expectedISøknadKontrakt));
+        // expect(JSON.stringify(formatert)).toEqual(JSON.stringify(expectedISøknadKontrakt));
     });
     it('case 4', async () => {
-        const { input: iSøknad, output: expectedISøknadKontrakt } = testdata4;
+        // const { input: iSøknad, output: expectedISøknadKontrakt } = testdata4;
+        const { input: iSøknad } = testdata4;
         spyOnUseApp(iSøknad);
         const { result } = renderHook(() => useSendInnSkjema(), {
             wrapper: TestProvidereMedEkteTekster,
         });
 
         const [_, formatert]: [boolean, ISøknadKontraktV8] = await result.current.sendInnSkjemaV8();
         expect(erGyldigISøknadKontrakt(formatert)).toBeTruthy();
-        expect(JSON.stringify(formatert)).toEqual(JSON.stringify(expectedISøknadKontrakt));
+        // expect(JSON.stringify(formatert)).toEqual(JSON.stringify(expectedISøknadKontrakt));
     });
     it('case 5', async () => {
-        const { input: iSøknad, output: expectedISøknadKontrakt } = testdata5;
+        // const { input: iSøknad, output: expectedISøknadKontrakt } = testdata5;
+        const { input: iSøknad } = testdata5;
         spyOnUseApp(iSøknad);
         const { result } = renderHook(() => useSendInnSkjema(), {
             wrapper: TestProvidereMedEkteTekster,
         });
 
         const [_, formatert]: [boolean, ISøknadKontraktV8] = await result.current.sendInnSkjemaV8();
         expect(erGyldigISøknadKontrakt(formatert)).toBeTruthy();
-        expect(JSON.stringify(formatert)).toEqual(JSON.stringify(expectedISøknadKontrakt));
+        // expect(JSON.stringify(formatert)).toEqual(JSON.stringify(expectedISøknadKontrakt));
```

Comment:
Hva er grunnen til at vi bypasser disse testene?

Reply:
Input og output fra testdata-filene tar i bruk lokale tekster for å sjekke at det som renderes stemmer overens med forventet output. Men siden vi bruker tekster fra Sanity, så er ikke de lokale tekstene relevante lengre.

Reply:
I see. Burde vi da kanskje fjerne koden du har kommentert ut fremfor å kommentere ut? 🤔

Reply:
Jeg kommenterte ut koden fordi det var slikt det ble gjort i KS for de samme testene, men jeg tror det er bedre å bare fjerne den kommenterte koden helt, så jeg gjør det 👍